provider/ec2/ebs.go

```diff
@@ -38,17 +38,24 @@ const (
 	//   "gp2" for General Purpose (SSD) volumes
 	//   "io1" for Provisioned IOPS (SSD) volumes,
 	//   "standard" for Magnetic volumes.
+	//   see volumes types below for more.
 	EBS_VolumeType = "volume-type"
 
 	// EBS_IOPS is the number of I/O operations per second (IOPS) per GiB
-	// to provision for the volume. Only valid for Provisioned
-	// IOPS (SSD) volumes.
+	// to provision for the volume. Only valid for io1 io2 and gp3 volumes.
 	EBS_IOPS = "iops"
 
+	// EBS_Throughput is the max transfer troughput for gp3 volumes.
+	EBS_Throughput = "throughput"
```

Comment:
Can someone get a list of these options or help on them using the CLI? I'm guessing not, but wondering how people would find out about them (before we update the docs).

Reply:
No, we need to work on this with config v2 :)
These things really should dynamically be exposed based on context (model/controller/cloud).